app.js

```diff
@@ -0,0 +1,65 @@
+const bodyParser = require('body-parser');
+const express = require('express');
+const app = express();
+const ldKeys = require('lodash/keys');
+app.set('views', __dirname + '/views');
+app.set('view engine', 'nunjs');
+app.use(bodyParser());
+const baseUtil = require('./base');
+
+const {WebsocketView} = require('./socketview');
+
+const server = require('http').Server(app);
+const io = require('socket.io')(server);
+const { PlanetWars, MAPS, AIS } = require('./game');
+
+const games = {};
+app.get('/', function(req, res) {
+  res.render('index', {
+    'map_names': ldKeys(MAPS),
+    'ai_names': ldKeys(AIS)
+  });
+});
+app.get('/game/:gameID', function(req, res) {
+  if (games[req.params.gameID]) {
+    res.render('game', {
+      gameID: req.params.gameID
+    });
+  } else {
+    res.redirect('/');
+  }
+});
+app.use('/static', express.static('static'));
```

Comment:
`app.use` should be above the `app.get`/`app.post` and they all should have empty lines around them.